test/Unit/TestAbstract.php

```diff
@@ -1,13 +1,14 @@
 <?php
-declare(strict_types = 1);
+declare(strict_types=1);
 
 namespace YapepBase\Test\Unit;
 
-use Mockery;
 use PHPUnit\Framework\TestCase;
 use YapepBase\Application;
 use YapepBase\DependencyInjection\Container;
 use YapepBase\DependencyInjection\PimpleContainer;
+use Mockery;
```

Comment:
Do you need this here?

Reply:
Don't like to \Mockery and all the other usages.